cni/pkg/install/kubeconfig.go

```diff
@@ -120,6 +120,11 @@ func createKubeconfigFile(cfg *config.InstallConfig, saToken string) (kubeconfig
 		return "", err
 	}
 
+	// When using Multus, the net.d dir might not exist yet, so we must create it
+	if err := os.MkdirAll(cfg.MountedCNINetDir, os.FileMode(0o755)); err != nil {
```

Comment:
Shouldn't we rather create this directory in the container image?

Reply:
No, this is a directory on the host node. It's mounted in the container, so the install-cni container can install the cni plugin onto the node.

Reply:
Ok, I understand.